README.md

```diff
@@ -104,4 +104,6 @@ NOTE: To update the component status:
 | Toggle Icon Button | [XD](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/d022d8af-22f4-4bf2-981c-1dc0c61afece/) | | [:white_check_mark: - SB](https://ni.github.io/nimble/storybook/?path=/story/toggle-button--icon-button) | :white_check_mark: | :white_check_mark: |
 | Toolbar | |  [Issue](https://github.com/ni/nimble/issues/411) | [:white_check_mark: - SB](https://ni.github.io/nimble/storybook/?path=/story/toolbar--toolbar) | :white_check_mark: | :white_check_mark: |
 | Tooltip | [XD](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/044414d7-1714-40f2-9679-2ce2c8202d1c/) |  [Issue](https://github.com/ni/nimble/issues/309) | [:warning: - SB](https://ni.github.io/nimble/storybook/?path=/docs/tooltip--tooltip) | :warning: | :warning: |
-| Tree View | | | [:white_check_mark: - SB](https://ni.github.io/nimble/storybook/?path=/docs/tree-view--tree-view) | :white_check_mark: | :white_check_mark: |
+| Tree View | [XD](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/ec5a855a-4174-46ad-947c-3931bbf3e32d/) | | [:white_check_mark: - SB](https://ni.github.io/nimble/storybook/?path=/docs/tree-view--tree-view) | :white_check_mark: | :white_check_mark: |![image]
```

Comment:
@nate-ni did you make these changes by updating the spreadsheet? When updating the spreadsheet it should spit out the updated table. I think the trailing `![image]` should not be there and is either a bug in the spreadsheet or a copy-paste error.

Reply:
Sorry @rajsite , just saw this. Yes, it is a copy paste error. Whenever I copy from the spreadsheet and paste into the MD it adds some image data at the end. I did remove part of the unintended image data that gets pasted but it looks like I missed that part.

Here is what it typically looks like before I "fix" it. Not sure why it does that, I'm just copying and pasting from Excel to here.

![image](https://user-images.githubusercontent.com/109235103/192883573-05a4a36e-17d5-4add-9d8f-0b03f5844e0d.png)